WordPress/gutenberg
# Reusable Blocks: Fix Server side rendering

The server-side rendering of reusable blocks was not working as expected because the attribute "ref" was dropped because it was a number instead of a string.

This PR changes the ref type to "number" which fixes the issue.

**Testing instructions**

 - Create a reusable block
 - Check that it show up properly on the frontend
